packages/graphiql-react/src/schema.tsx

```diff
@@ -168,6 +168,8 @@ export function SchemaContextProvider(props: SchemaContextProviderProps) {
       }
 
       setIsFetching(true);
+      // clear any previous fetch errors
```

Comment:
I don't think this comment is necessary, it's very clear what the following line does.
```suggestion
```